polars/polars-core/src/frame/mod.rs

```diff
@@ -313,6 +313,23 @@ impl DataFrame {
     }
 
     /// Get a reference to the DataFrame columns.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use polars_core::df;          // or "use polars::df"
+    /// use polars_core::prelude::*;  // or "use polars::prelude::*"
+    ///
+    /// fn example() -> Result<()> {
+    ///     let df: DataFrame = df!("Name" => &["Adenine", "Cytosine", "Guanine", "Thymine"],
+    ///                             "Symbol" => &["A", "C", "G", "T"])?;
+    ///     let columns: &Vec<Series> = df.get_columns();
+    ///
+    ///     assert_eq!(df.shape().1, columns.len());
```

Comment:
Maybe assert something that's more related to the `get_columns` output? 

Like

```rust
assert_eq!(columns, &["Name", "Symbol"]);
```